weblab_cg/_printer.py

```diff
@@ -61,40 +61,14 @@ class Printer(sympy.printing.printer.Printer):
         A function that converts derivatives to strings.
 
     """
-    # List of functions handled by python's `math` module
-    _math_functions = {
-        'acos',
-        'acosh',
-        'asin',
-        'asinh',
-        'atan',
-        'atan2',
-        'atanh',
-        'ceil',     # called ceiling in sympy
-        'cos',
-        'cosh',
-        'exp',
-        'expm1',
-        'factorial',
-        'floor',
-        'log',
-        'log10',
-        'log1p',
-        'log2',
-        'sin',
-        'sinh',
-        'tan',
-        'tanh',
-    }
-
-    # List of custom defined functions we are allowed to output
-    _custom_functions = set()
+    # Dictionary of functions we can handle and their corresponding names to be generated
+    _function_names = {}
 
     def __init__(self, symbol_function=None, derivative_function=None):
         super(Printer, self).__init__(None)
 
         # Prefix for functions
-        self._prefix = 'math.'
+        self._prefix = ''
```

Comment:
This still needs to be "math." to make it work for sqrt etc.

An alternative would be to
1. Remove self._prefix
2. Add "sqrt" and co into the list of functions
3. Make functions that now use `self._prefix` lookup the correct name for "sqrt" in the list of functions